astronomer/providers/dbt/cloud/operators/dbt.py

```diff
@@ -0,0 +1,80 @@
+import time
+from typing import Any, Dict, Optional
+
+from airflow import AirflowException
+from airflow.providers.dbt.cloud.hooks.dbt import DbtCloudHook
+from airflow.providers.dbt.cloud.operators.dbt import DbtCloudRunJobOperator
+from airflow.utils.context import Context
+
+from astronomer.providers.dbt.cloud.triggers.dbt import DbtCloudRunJobTrigger
+
+
+class DbtCloudRunJobOperatorAsync(DbtCloudRunJobOperator):
+    """
+    Executes a dbt Cloud job asynchronously. Trigger the dbt cloud job via worker to dbt and with run id in response
+    poll for the status in trigger.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:DbtCloudRunJobOperator`
+
+    :param dbt_cloud_conn_id: The connection ID for connecting to dbt Cloud.
+    :param job_id: The ID of a dbt Cloud job.
+    :param account_id: Optional. The ID of a dbt Cloud account.
+    :param trigger_reason: Optional. Description of the reason to trigger the job.
+    :param steps_override: Optional. List of dbt commands to execute when triggering the job instead of those
+        configured in dbt Cloud.
+    :param schema_override: Optional. Override the destination schema in the configured target for this job.
+    :param wait_for_termination: This Flag is disabled in this async operator.wait_for_termination
+        will be always set True
```

Comment:
why is wait_for_termination True? shouldn't it be set to False for async?

Reply:
For async it should be True